Client/Frontend/Browser/ReaderModeBarView.swift

```diff
@@ -8,6 +8,32 @@ import SnapKit
 
 enum ReaderModeBarButtonType {
     case MarkAsRead, MarkAsUnread, Settings, AddToReadingList, RemoveFromReadingList
+
+    private var description: String {
```

Comment:
Would you mind changing this into `localizedDescription` instead? I would like to keep `description` reserved for what the `Printable` protocol is supposed to do. (Which is a non-user-facing technical description of objects)